exporter/clickhouseexporter/exporter_logs.go

```diff
@@ -238,17 +246,3 @@ func renderCreateLogsTableSQL(cfg *Config) string {
 func renderInsertLogsSQL(cfg *Config) string {
 	return fmt.Sprintf(insertLogsSQLTemplate, cfg.LogsTableName)
 }
-
-func doWithTx(_ context.Context, db *sql.DB, fn func(tx *sql.Tx) error) error {
```

Comment:
this was not the proper way, they were using a tx but they were using an insert per row which is not correct, it's one  prepared insert and then you append each row to the batch using `Exec()`